neqo-http3/src/connection_server.rs

```diff
@@ -235,22 +346,37 @@ impl Http3ServerHandler {
         now: Instant,
         stream_id: u64,
         buf: &mut [u8],
+        wt: bool,
```

Comment:
```suggestion
        webtransport: bool,
```

... though I don't think that you want this.  More below.